src/index.js

```diff
@@ -83,6 +71,8 @@ pBottom.classList.add('p-bottom');
 pBottom.innerHTML = 'Clear all completed';
 listItemBottom.appendChild(pBottom);
 
+pBottom.addEventListener('click', clearAllComplele);
+
 setTimeout(() => {
   myWrapper.appendChild(listItemBottom);
 }, 200);
```

Comment:
- [OPTIONAL] This operation is proving expensive and sometimes it leads to abnormal/unexpected behavior by breaking the UI. See the screenshot below:
![unexpected-behaviour](https://user-images.githubusercontent.com/45002654/134317240-46de2ab0-18e0-4992-86b9-ea6379202b85.PNG)

The `Clear All Completed button` gets positioned above the todo items. Ideally, this button should not be a part of the list. I propose you code it in the HTML inside your `<main></main>` tags after the list,  so that the UI can maintain it's look and feel after any operation.